services/mgmt/cosmosdb/src/package_2021_07_preview/models.rs

```diff
@@ -1742,10 +1742,6 @@ pub mod restorable_sql_container_properties {
         use super::*;
         #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
         pub struct Container {
-            #[serde(flatten)]
-            pub sql_container_resource: SqlContainerResource,
-            #[serde(flatten)]
-            pub extended_resource_properties: ExtendedResourceProperties,
```

Comment:
This shouldn't be dropped. Looks like the `allOf` may not be populated for local models.
```
            "database": {
              "type": "object",
              "description": "Cosmos DB SQL database resource object",
              "properties": {
                "_colls": {
                  "type": "string",
                  "readOnly": true,
                  "description": "A system generated property that specified the addressable path of the collections resource."
                },
                "_users": {
                  "type": "string",
                  "readOnly": true,
                  "description": "A system generated property that specifies the addressable path of the users resource."
                },
                "_self": {
                  "type": "string",
                  "readOnly": true,
                  "description": "A system generated property that specifies the addressable path of the database resource."
                }
              },
              "allOf": [
                {
                  "$ref": "../../preview/2021-07-01-preview/cosmos-db.json#/definitions/SqlDatabaseResource"
                },
                {
                  "$ref": "../../preview/2021-07-01-preview/cosmos-db.json#/definitions/ExtendedResourceProperties"
                }
              ]
            }
```
specification\cosmos-db\resource-manager\Microsoft.DocumentDB\preview\2021-07-01-preview\restorable.json